sympy/stats/stochastic_process_types.py

```diff
@@ -231,9 +232,6 @@ class DiscreteMarkovChain(DiscreteTimeStochasticProcess):
     >>> P(Eq(Y[3], 2), Eq(Y[1], 1)).round(2)
     0.36
     """
-
-    is_markov = True
```

Comment:
Why this flag is removed? Any specific reasons?

Reply:
it wasn't required